### PR TITLE
Fix directory creation bug in download_bundle.

### PR DESCRIPTION
**Pull request by douglaseck**
Unless tmp/bundles happens to exist already, this code fails. I guess not too many people are using the Performance_RNN jupyter notebook.

### CODE REVIEW

File: magenta/music/notebook_utils.py [hunk 1]
```diff
@@ -132,11 +132,15 @@ def _sequence_to_pandas_dataframe(sequence):
...
 
+  Target directory target_dir will be created if it does not already exist.
+
   Args:
      bundle_name: A string Magenta bundle name to download.
      target_dir: A string local directory in which to write the bundle.
      force_reload: A boolean that when True, reloads the bundle even if present.
   """
+  if not os.path.exists(target_dir):
+    os.makedirs(target_dir)
```
Comment (cghawthorne) [inline magenta/music/notebook_utils.py]: Should just call:
```
tf.gfile.MakeDirs(target_dir)
```

The gfile version has better internal compatibility and will just return if the directory already exists, so no need to do the os.path.exists check.

**Comment by douglaseck** [comment]
I fixed a bad email address in my .gitconfig.  You can ignore the CLA message.

**Comment by douglaseck** [comment]
PTAL

**Comment by douglaseck** [comment]
Adding comment to trigger cla check

**Pull request merged**